packages/server/package.json

```diff
@@ -88,11 +88,13 @@
     "knex": "^0.14.2",
     "lodash": "^4.17.4",
     "minilog": "^3.1.0",
+    "mobile-detect": "^1.4.1",
```

Comment:
Please remove this dependency and use 
```js
import {Platform} from 'react-native';

if (Platform.OS === '...') {
}
``` 
instead.
`react-native` is automatically linked to `react-native-web` on Web front-end and on backend and this approach will work everywhere in our code.

Reply:
I used Platform.OS, but it always return 'web'.

Reply:
@sergei-gilevich Yes, for `server` and `web` Platform.OS will return `web`, for `android` it will return `android` and for `ios` it will return `ios`